bin.js

```diff
@@ -31,3 +34,12 @@ function getConfig() {
   }
 }
 
+function getRules(set) {
+  var configFileName = set ? ('eslint-config-' + set) : null
```

Comment:
Ah. I see. Yes, if your config has `extends: "eslint:recommended"` all of the rules from it will be in the output of `getConfigForFile`. If you don't want that, we don't provide anything like that. Not sure what would be the best approach in that case. You could still use `getConfigForFile` and then just compare it against local config file, and report missing keys